### PR TITLE
add theme for banner notification

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
Adds theme for banner notification
#### What testing has been done on this PR?
DS site
#### Any background context you want to provide?
view changes https://deploy-preview-2363--keen-mayer-a86c8b.netlify.app/templates/global-banner-notifications?q=glo
#### What are the relevant issues?
completes banner notification issue
#### Is this change backward compatible or could it be a breaking change for the official HPE theme?
backward compatible
#### How should this PR be communicated in the release notes?
yes

### CODE REVIEW

File: src/js/themes/hpe.js [hunk 1]
```diff
@@ -833,6 +833,59 @@ export const hpe = deepFreeze({
...
+      round: 'xsmall',
+    },
+    global: {
+      container: {
+        round: 'none',
+      },
+    },
+    critical: {
+      background: 'validation-critical',
+      toast: {
```
Comment (taysea) [inline src/js/themes/hpe.js:851]: I think in latest grommet stable we now have a defined color for notification[status]global.background -- which we might need to override to ensure out validation colors are coming through.

Can you double check that?

Comment (britt6612) [reply src/js/themes/hpe.js:851]: yes Ill check now

Comment (britt6612) [reply src/js/themes/hpe.js:851]: yes the critical is coming through so we don't need that just the rest

Comment (britt6612) [reply src/js/themes/hpe.js]: looks like we dont need `toast` background colors either

**Review by jcfilben — APPROVED** [review]
Looks good!

Side note: I think we need to updated the grommet-site docs to include `notification.global`

**Pull request merged**